plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -51,6 +51,7 @@
     private boolean caseInsensitiveNameMatching;
     private Duration viewsCacheTtl = new Duration(15, MINUTES);
     private Duration serviceCacheTtl = new Duration(3, MINUTES);
+    private Duration metadataCacheTtl = new Duration(1, MINUTES);
```

Comment:
Default should be disabled IMO. Metadata caching has a tradeoff regarding changes made in source not being immediately visible to users.

Also note that the performance penalty only happens when case-insensitive-name-matching is enabled - which is not enabled by default. So caching by default only makes sense when case-insensitive-name-matching is enabled.